File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.cpp
class Solution {
public:
    int mostWordsFound(vector<string>& sentences) {
        int n = sentences.size();
        int count=0;
        int highcount=0;
        for(int i =0;i<n;i++)
        {
            const string&str = sentences[i];
            count=0;
            
            for(int j=0;j<str.length();j++)
            {
                if(str[j] == ' ')
                {
                    count++;
                }
            }
            count++;  
            highcount = max(highcount,count);
        }
        return highcount;        
    }
};

File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.py
class Solution(object):
    def mostWordsFound(self, sentences):
        count=0
        highcount=0
        n=len(sentences)
        for i in range(n):
            words=sentences[i].split()
            count=len(words)
            highcount=max(count,highcount)
            
        return highcount
            
        """
        :type sentences: List[str]
        :rtype: int
        """
        